lib/features/family/features/impact_groups/widgets/dialogs/preferred_church_outcome_dialog.dart

```diff
@@ -21,3 +22,22 @@ Future<void> showPreferredChurchSuccessDialog(
     ],
   );
 }
+
+Future<void> showPreferredChurchErrorDialog(
+  BuildContext context,
+) async {
+  return FunModal.showAsDialog(
+    context,
+    title: 'Oops, something went wrong...',
+    buttons: [
+      FunButton(
+        text: 'Retry',
+        onTap: () => context.pop(),
```

Comment:
Just to be sure I don't see any retry happening in the logic. I assume this means this dialog is shown on the selectionpage itself and you need to click the green button again?